docs/wxm-devices/D1-bundle/wb1200-introduction.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: D1-bundle-introduction
-title: WeatherXM D1 Bundle (WB1200) Introduction
+title: WeatherXM D1 Bundle Introduction
```

Comment:
This was correct as it was before, please revert it to:
WeatherXM D1 Bundle (WB1200) Introduction